INSTALL.md

```diff
@@ -141,65 +141,62 @@ packages necessary for building .debs, so be prepared to type your password for
 
 <a name="osx" />
 ##OS X
+Before we start, be aware that this section had a large guide that succeeded in only confusing.
+I've thrown it entirely out for something actually useful and helpful.
 
-###OSX Easy Install
+Also, if you want to use qTox download it by clicking the download button on tox.im, compiling is a pain
+that I don't want you to ever deal with and your finished project will have a couple issues.
 
-Since https://github.com/ReDetection/homebrew-qtox you can easily install qtox with homebrew 
-```bash
-brew install --HEAD ReDetection/qtox/qtox
-```
+Compiling qTox on OS X for development requires 3 tools, [Xcode](https://developer.apple.com/xcode/) and [Qt 5.4+](http://www.qt.io/qt5-4/), and [homebrew](http://brew.sh).
 
+###Required tools
 
-###OSX Full Install Guide
+First, let's install the dependencies
+* ```brew install git wget```
+* ``git clone https://github.com/tux3/qTox``
+* ```cd qTox```
 
-This guide is intended for people who wish to use an existing or new ProjectTox-Core installation separate to the bundled installation with qTox, if you do not wish to use a separate installation you can skip to the section titled 'Final Steps'.
+###Libraries required to compile
 
-Installation on OSX, isn't quite straight forward, here is a quick guide on how to install;
+Now we are in the qTox folder and need our library dependencies to actually build it.
 
-Note that qTox now requires OpenCV and OpenAL for video and audio.
+I've taken the time to prepare them automatically with our CI system so if you ever have issues redownload them.
 
-The first thing you need to do is install ProjectTox-Core with a/v support. Refer to the INSTALL guide in the PrjectTox-Core github repo.
+* ```wget https://jenkins.libtoxcore.so/job/qTox%20OS%20X/lastSuccessfulBuild/artifact/dep.zip```
```

Comment:
@tux3 I really can't reproduce making the folder out of Jenkins in any sane way that doesn't totally fuck up the system you're doing it on. 

Regardless, I made a note on doing it without our binaries.